scripts-dev/schema_versions.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# Copyright 2023 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import json
+import sys
+from collections import defaultdict
+from typing import Any, Dict, Iterator, Optional, Tuple
+
+import git
```

Comment:
This isn't installed by default we could either import our virtualenv or just directly install gitpython in CI for the docs jobs?

Reply:
It's in dev-dependencies, isn't it?

Reply:
Yes, but we don't install that for the build docs GHA. (Since it uses gitbook, which is Rust)

Reply:
ah I see. Bleh. Pity you can't just use one dev-dependencies install and then fork off all your CI jobs from that. Anyway, feel free to just directly install gitpython (i.e. do the minimal pain thing)